com.kristianduske.TrenchBroom.json

```diff
@@ -0,0 +1,144 @@
+{
+    "app-id": "com.kristianduske.TrenchBroom",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "trenchbroom.sh",
+    "rename-desktop-file": "trenchbroom.desktop",
```

Comment:
No need to rename the desktop file if you install it directly.

Reply:
Eh? Ok, I deleted that line.